exec/java-exec/src/main/java/org/apache/drill/exec/expr/fn/impl/StringFunctions.java

```diff
@@ -786,10 +784,11 @@ public void setup() {
 
     @Override
     public void eval() {
-      out.buffer = buffer = buffer.reallocIfNeeded(input.end - input.start);
+      byte[] result = org.apache.drill.exec.expr.fn.impl.StringFunctionHelpers.initCap(input.start, input.end, input.buffer);
```

Comment:
The earlier functions handle the DrillBuf to string conversion and the String to byte[] conversion. Might want to do that here, changing the initCap() method to:

```
String initCap(String input)
```

Reply:
@paul-rogers review, please